### ISSUE TITLE
Hi, I'm having some issues with many channels, after the migration to the new server I have a lot of instabilities and buffering problems.

### ISSUE THREAD

**Issue by NyanCat6886** [luongz/iptv-jp#141]
Hi, I'm having some issues with many channels, after the migration to the new server I have a lot of instabilities and buffering problems.
What can I do?

_Originally posted by @NyanCat6886 in https://github.com/luongz/iptv-jp/discussions/117#discussioncomment-11574667_

**Comment by weldon1045** [comment]
MX1 Test seems to be down.

**Comment by luongz** [comment]
> Dear Luong,
> Being in the position of an infrastructure and hosting provider, it is our obligation to ensure a fair allocation of the available bandwidth 
amongst and in favour of all Contabo customers likewise. To fulfill this requirement, various shaping mechanisms have been implemented, which regulate such fair distribution of bandwidth at runtime.
>
> While this aforementioned instrument already contributes essentially towards our aim to guarantee a fair bandwidth allocation, we found that it may not suffice in every situation and that it requires an additional tool to regulate extraordinary bandwidth consumption which comes into effect, when such is gauged.
> Your VPS 1 SSD (no setup) is equipped with a 200 Mbit/s port by default.
>
> During the last 10 days your VPS 1 SSD (no setup) had been consuming an average outgoing bandwidth of 137 Mbit/s. 
>As mentioned in the regulations on our website, we are forced to throttle the port speed of your instance if the outgoing bandwidth exceeds the bandwidth package for 10 days in a row. To ensure the commitment of ours, to guarantee a fair share of the available bandwidth between all users, we just switched the port of your server to 100 Mbit/s.

_Originally posted by @luongz in https://github.com/luongz/iptv-jp/discussions/117#discussioncomment-11361671_

There is only one way to solve the buffer issues: rent a better server. (100$ per month is very expensive to me lol)

**Comment by NyanCat6886** [comment]
@luongz Oh, I had no idea.
That's unfortunate.
Well I'm not an expert in the subject, but I guess if other people use their own servers along with yours to host the channels, if one server is not working well, the service could run in another server with better speed. Again, I don't know if that's possible but if that works, it could reduce the buffer problems.

**Comment by carmagnolle** [comment]
you guys know what to do https://ko-fi.com/dinhluong

this is a public project and if a small percentage of us managed to send a "coffee" each towards it that combined would pay for better servers for months, so definitely consider doing it

**Comment by weldon1045** [comment]
MX1 test signal seems to be broken right now.

**Comment by DimitarCC** [comment]
The main problem is that currently vthanh channels are not watchable... So no point to have them in the playlist...
Otherwise i do encourage people to support the project... If we all pay 5$ per month for sure we can reach 100$..

**Comment by DimitarCC** [comment]
> MX1 test signal seems to be broken right now.

At-x and bs11 test signals also not working from days..

**Comment by nazonomi** [comment]
MX1 has the same channel on the vthanh server, so you can watch it there. 

The vthanh server is currently available. Why don't you take another look at your environment?

**Comment by weldon1045** [comment]
Tested VLC on PC. Tivimate on Android TV Box, VLC and other IPTV player on iPad and all of them have a terrible result of constant buffering plus VLC straight up skipping the channel.

The test channels don't have the same issues the vthanh channels currently have.

**Comment by DimitarCC** [comment]
> MX1 has the same channel on the vthanh server, so you can watch it there. 
> 
> The vthanh server is currently available. Why don't you take another look at your environment?

Seems no bandwidth so vthanh are unusable anyway even when available. A lot of shuttering and not watchable.

**Comment by DimitarCC** [comment]
> Tested VLC on PC. Tivimate on Android TV Box, VLC and other IPTV player on iPad and all of them have a terrible result of constant buffering plus VLC straight up skipping the channel.
> 
> The test channels don't have the same issues the vthanh channels currently have.

Yes. But they are on another server. But @luongz said he can not add more channels as a test on that server.

**Comment by luongz** [comment]
I will temporarily add some terrestrial channels to the Test group. Still, I need to shut down some Test channels first (TBS Ch1/2, Space Shower, and JSport to save bandwidth), and keep it running until I find a server that I can afford (and I need suggestions for hosting provider too - my budget is 50$/month)

**Comment by luongz** [comment]
Temporary added: NHK E, TBS, NTV, Fuji, TV Asahi, TV Tokyo, MX1
and temporarily shut down: Pigoo, JS1/2/3/4, MX2, Space Shower, TBS Ch 1/2.

**Comment by DimitarCC** [comment]
> Temporary added: NHK E, TBS, NTV, Fuji, TV Asahi, TV Tokyo, MX1
> and temporarily shut down: Pigoo, JS1/2/3/4, MX2, Space Shower, TBS Ch 1/2.

Playlist not updated yet seems 😉

**Comment by luongz** [comment]
Updated.

**Comment by dingusmcdingus** [comment]
AND your streams are now stuttering. The NHK G stream worked great before more channels were added, but now it's just as laggy as vthanh's streams. I wonder how many streams the server can handle before struggling...

**Comment by DimitarCC** [comment]
@luongz You said you need 100$/mo for good server that will not make shuttering...But your budget is 50$/mo.
OK. I can sponsor you with $10/mo if you agree. (the only requirement from my side is ofcource to have japanese channels without shuttering since my wife is japanese and need those channels)
If some other people here agree to give $10/mo we can easily reach $100/mo. Right?
Or can you find a hosting for $60/mo?

**Comment by carmagnolle** [comment]
> @luongz You said you need 100$/mo for good server that will not make shuttering...But your budget is 50$/mo. OK. I can sponsor you with $15/mo if you agree. (the only requirement from my side is ofcource to have japanese channels without shuttering since my wife is japanese and need those channels) If some other people here agree to give $10-15/mo we can easily reach $100/mo. Right? Or can you find a hosting for $65/mo?

I can guarantee to donate at least once every month, I'm "carma" on Ko-fi

**Comment by DimitarCC** [comment]
So is pretty much doable to reach $100 seems... What will say @luongz  ;)

**Comment by dingusmcdingus** [comment]
update: the streams seem to run great now!
i don't know why they were so stuttery yesterday, maybe it wasn't even the streams but rather my internet...

**Comment by DimitarCC** [comment]
No. It was the streams. It was shuttering to me too yesterday...

**Comment by dingusmcdingus** [comment]
> No. It was the streams. It was shuttering to me too yesterday...

and it's stuttering again...
also for some reason the nippon tv stream airs tv asahi

**Comment by DimitarCC** [comment]
> > No. It was the streams. It was shuttering to me too yesterday...
> 
> and it's stuttering again... also for some reason the nippon tv stream airs tv asahi

No chance to be fixed before a new server is bought. When there are many people watching there is no enough bandwidth. Thats why streams are shuttering...

**Comment by TheAntinormal** [comment]
Why not use Cloudflare CDN to cache every *.ts segments that are available on your domain? @luongz 

It's free, fast, and would help a lot with stability as the only requests that'll still be served by your server will be the playlists and some segments (until they're cached), the other segments could then be fetched from the CDN? 

If that's already the case, then nevermind.

**Comment by luongz** [comment]
Until now, I have not found a suitable server provider.

**Comment by dingusmcdingus** [comment]
perhaps you should shutdown some of the terrestrial channels

**Comment by dingusmcdingus** [comment]
heck you should just revert back to just nhk g

**Comment by namsse** [comment]
ntv test and asahi test broadcast the same video.

**Comment by wing3830** [comment]
I can't watch NTV and ETV... When will this problem be fixed?

**Comment by weldon1045** [comment]
Tokyo MX test is down

**Comment by luongz** [comment]
VThanh channels have been moved to a new server. 
Can everyone check to see if there are any problems?

**Comment by kkdexz** [comment]
> VThanh channels have been moved to a new server. Can everyone check to see if there are any problems?
> 
> (don't worry, test channels will be back soon)

Not working on my end atm. Says the server is unreachable for vthanh links. "Not found" for test server.

Maybe too soon? Might work in a bit, after migration is complete perhaps?

**Comment by DimitarCC** [comment]
Not working for now on my side too....

**Comment by nekoshop** [comment]
luongz remove the data from my server, so its not working now 😅

**Comment by DimitarCC** [comment]
> VThanh channels have been moved to a new server. Can everyone check to see if there are any problems?
> 
> (don't worry, test channels will be back soon)

Well but he said to test so i am confused....

**Comment by luongz** [comment]
Update your playlist and enjoy.

(it seems we need more donations to maintain this server :D)

**Comment by kkdexz** [comment]
> Update your playlist and enjoy.
> 
> (it seems we need more donations to maintain this server :D)

Yes, thank you very much! All working flawlessly once again (like before). Oh how I wished for this day to come lol.

Here's hoping it lasts a while this time. Definitely planning to donate very soon.
